internal/handler/commands/say.go

```diff
@@ -7,7 +7,8 @@ import (
 func (h BotCommandHelp) Say(request BotCommand) (response HelpResponse) {
     response.Help = "Give Bender a line of text to say in a channel."
 
-    response.Description = "Cause the bot to say something in a channel. Usage: '!say in {channel} {text}'"
+    response.Description = "Cause the bot to say something in a channel. " +
```

Comment:
I think you need a `\n` at the end of the first line or it will still print out on one line